src/Data/Nat/Divisibility.agda

```diff
@@ -179,6 +182,33 @@ module _ where
       q * (i * (suc k))  ≡⟨ sym (*-assoc q i (suc k)) ⟩
       (q * i) * (suc k)  ∎))
 
+  m%n≡0⇒n∣m : ∀ m n → m % (suc n) ≡ 0 → suc n ∣ m
+  m%n≡0⇒n∣m m n eq = divides (m div (suc n)) (begin
+    m                                     ≡⟨ a≡a%n+[a/n]*n m n ⟩
+    m % (suc n) + m div (suc n) * (suc n) ≡⟨ cong₂ _+_ eq refl ⟩
+    m div (suc n) * (suc n)               ∎)
+
+  n∣m⇒m%n≡0 : ∀ m n → suc n ∣ m → m % (suc n) ≡ 0
+  n∣m⇒m%n≡0 m n (divides v eq) = begin
+    m           % (suc n) ≡⟨ cong (_% (suc n)) eq ⟩
+    (v * suc n) % (suc n) ≡⟨ kn%n≡0 v n ⟩
+    0                     ∎
+
+  m%n≡0⇔n∣m : ∀ m n → m % (suc n) ≡ 0 ⇔ suc n ∣ m
+  m%n≡0⇔n∣m m n = record
+    { to   = PropEq.→-to-⟶ (m%n≡0⇒n∣m m n)
+    ; from = PropEq.→-to-⟶ (n∣m⇒m%n≡0 m n)
+    }
+
```

Comment:
I have learned today that this one can be constructed using the function [`equivalence`](https://agda.github.io/agda-stdlib/Function.Equivalence.html#804)

Reply:
Neat. Now if only I can find all the previous places where I've seen this pattern!

Reply:
Should I open a new issue with the result of `find . -name "*.agda" | xargs grep "from.*=.*→-to-⟶"`? :D